harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
That's good to know - I'm not aware of any workable in-browser tests for test262 yet, or else I'd be running them on the es6-shim. Pointers commented on https://github.com/paulmillr/es6-shim/issues/278 would be appreciated.